hw/mcu/nordic/nrf5340/tfm/src/tfm.c

```diff
@@ -37,6 +37,26 @@ tfm_uicr_otp_read(uint8_t n, uint32_t *ret)
     return err;
 }
 
+int SECURE_CALL
+tfm_uicr_otp_write(uint8_t n, uint32_t val)
+{
+    int err = 0;
+
+    if (n >= 192) {
+        err = TFM_ERR_INVALID_PARAM;
+    } else if ((n < MYNEWT_VAL(TFM_UICR_OTP_MIN_ADDR)) ||
```

Comment:
As those mynewt vals are now used also for write, their help messages should also be updated

Reply:
done